pkg/cluster/wait.go

```diff
@@ -51,6 +52,10 @@ func WaitFor(ctx context.Context, client kubernetes.Reader, cluster *anywherev1.
 
 		observedGeneration := c.Status.ObservedGeneration
 		generation := c.Generation
+
+		log := logger.Get()
+		log.V(9).Info("cluster generation is %s and observedGeneration is %s", generation, observedGeneration)
```

Comment:
Does this work? this is a strctured logger, it takes key values, not a formatting sequence

Reply:
sorry I did not test this and completely missed this.